README.md

```diff
@@ -6,6 +6,7 @@
 
 [![Build Status](https://img.shields.io/travis/taichi-dev/taichi?logo=Travis&branch=master)](https://travis-ci.com/taichi-dev/taichi)
 [![Build Status](https://img.shields.io/appveyor/build/yuanming-hu/taichi?logo=AppVeyor)](https://ci.appveyor.com/project/yuanming-hu/taichi/branch/master)
+[![Codecov Status](https://codecov.io/gh/taichi-dev/taichi/branch/master/graph/badge.svg)](https://codecov.io/gh/taichi-dev/taichi)
```

Comment:
One way to make it clear that the cpp coverage is not included in this report yet might be using shields.io over the default badge?

```suggestion
[![Codecov Status](https://img.shields.io/codecov/c/github/taichi-dev/taichi?label=Python%20Code%20Coverage&logo=codecov)](https://codecov.io/gh/taichi-dev/taichi)
```
which results in:

![Codecov Status](https://img.shields.io/codecov/c/github/taichi-dev/taichi?label=Python%20Code%20Coverage&logo=codecov)